src/rtcModule/IRtcStats.h

```diff
@@ -34,6 +35,9 @@ struct Sample
             short width = 0;        // width of frame
             short height = 0;       // height of frame
             long bwav = 0;          // bandwidth available
+            long firtx = 0;
```

Comment:
Please, add some comment to this variable and the ones below

Reply:
Solved it